include/odp/api/spec/schedule.h

```diff
@@ -45,6 +46,53 @@ extern "C" {
  * Maximum schedule group name length in chars including null char
  */
 
+/**
+ * Schedule configuration options
+ */
+typedef struct odp_schedule_config_t {
+
+	/* Scheduler flow aware
+	 * Application can configure the scheduler to be flow aware or unaware.
+	 * When the scheduler is configured as flow aware the ORDERED and
+	 * ATOMIC schedule queue synchronization by the scheduler is performed
+	 * per flow within each queue.
+	 */
+	odp_support_t flow_aware;
+
+	/* Number of parallel flows per queue to be supported
```

Comment:
Why "parallel" here? Is this a synonym for independent? We already use parallel for a queue type so this may be a confusing reuse of the term.